Python.md

```diff
@@ -15,23 +20,63 @@ As most of the other projects written in Python, we use [PEP 8]
 (https://www.youtube.com/watch?v=wf-BqAjZb8M)" by Raymond Hettinger at PyCon 2015,
 which talks about the famous P versus NP problem.*
 
+If the project has a `pylintrc` file, you can use pylint to check for potential style violations:
+
+    pylint --rcfile=pylintrc
+
 The general idea is to make the code within a project consistent and easy to interpret (for humans).
 
+
 ### Docstrings
 
-Unless the function is easy to understand quickly, it should have a docstring
-describing what it does, how it does it, what the arguments are, and what the
-expected output is.
+Include docstrings in functions. This should say *what* the function does, its *inputs*,
+and *outputs* or possible *failure states* (exceptions).
 
-Most of our Python projects use "Google-style" docstrings:
+Unless otherwise stated, we use "Google-style" docstrings:
 https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments.
 
+For flask application docstrings for views are turned into API Documentation
+(See an example for AcousticBrainz at http://acousticbrainz.readthedocs.io/)
```

Comment:
Are they?

They are only if project uses Sphinx with a specific extension. I think it would be a good idea to make a subsection of this and describe the suggested environment for documenting web APIs with some guidelines on how we do this (if it differs from defaults required by Sphinx and autohttp.flask).